README.md

```diff
@@ -211,6 +211,21 @@ Mac users need to apply a workaround described in [About the Network](./docs/con
 In rare cases the port `8080` of the `winery` container may be inaccessible.
 In these cases it may help to change the port in the docker-compose file (e.g. to `8079:8080` [`<outside-port>:<container-port>`]) and use the config options described in [How to configure the OpenTOSCA UI to use a different Winery repository](./docs/advanced-how-to.md#how-to-configure-the-opentosca-ui-to-use-a-different-winery-repository) to point the OpenTOSCA ui to the new port for the Winery.
 
+### Http Certificates Cannot be Validated / Clocks Out of Sync / `apt update` fails
+
+Most noticable is the `apt update` command failing with errors like "Release file is not yet valid".
```

Comment:
Maybe add the actual error message also to ease finding the anaswer. Add something like:
> If you get something like `E: Release file for http://dl.google.com/linux/chrome/deb/dists/stable/Release is not valid yet (invalid for another 2h 45min 28s). Updates for this repository will not be applied.`...